algorithms/1-intro/fibonnaci_iterative.rb

```diff
@@ -0,0 +1,27 @@
+def fib(n)
+  if n == 0
+    return nil
+  elsif n == 1
+    puts 0
+  else
+    counter = 2
+    ten_returned = [0, 1]
+    while counter < n
+      next_number = ten_returned[-1] + ten_returned[-2]
+      ten_returned << next_number
+      counter += 1
```

Comment:
sorry! that's an old habit from before I learned how to use for loops in javascript.